Services/source/housekeeping/housekeeping_service.c

```diff
@@ -231,9 +231,16 @@ Result mock_everyone(All_systems_housekeeping *all_hk_data) {
 
     // Athena
     uint8_t i;
-    for (i = 0; i < 6; i++) {
+    for (i = 0; i < 2; i++) {
```

Comment:
This should be NUM_TEMP_SENSOR, not 2.

Reply:
Unless we don't want to use that macro because it's in another repo?

Reply:
It would be a good idea to use it